opentelemetry-instrumentation/src/opentelemetry/instrumentation/bootstrap.py

```diff
@@ -15,118 +15,19 @@
 # limitations under the License.
 
 import argparse
-import pkgutil
+import logging
 import subprocess
 import sys
-from logging import getLogger
 
-from opentelemetry.instrumentation.version import __version__ as version
-
-logger = getLogger(__file__)
-
-
-# A mapping of "target library" to "desired instrumentor path/versioned package
-# name". Used as part of the `opentelemetry-bootstrap` command which looks at
-# libraries used by the application that is to be instrumented, and handles
-# automatically installing the appropriate instrumentations for that app.
-# This helps for those who prefer to turn on as much instrumentation as
-# possible, and don't want to go through the manual process of combing through
-# the libraries their application uses to figure which one can be
-# instrumented.
-# NOTE: system-metrics is not to be included.
-def all_instrumentations():
```

Comment:
Seeing this code being removed makes me **very** happy 🥳

Reply:
I know!!